components/director/internal/open_resource_discovery/validation_test.go

```diff
@@ -1813,6 +1919,7 @@ func TestDocuments_ValidateAPI(t *testing.T) {
 				doc := fixORDDocument()
 				doc.APIResources[0].ResourceDefinitions = nil
 				doc.APIResources[0].Visibility = str.Ptr(open_resource_discovery.ApiVisibilityPrivate)
+				doc.APIResources[0].VersionInput.Value = "2.1.3"
```

Comment:
These version bumps are obsolete.

As this and the remaining similar test cases do not test the version bump logic, I think these bumps can be deleted as they do not change anything if the hash is not re-evaluated